golangci.yml

```diff
@@ -0,0 +1,29 @@
+run:
+  timeout: 10m
+  tests: true
+  skip-dirs:
+    - "storage/.*/sql/migrations"
```

Comment:
maybe more like:

```
skip-files:
-  ".*/bindata.go"
```

we used gobindata in other scenario  than sql migrations (i.e. cql migrations, frontend templating in acqusition etc...)

Reply:
Alright